File: calculadora.java
import java.util.Scanner;

public class Main{
	public static void main(String[]args) {
    
    
    Scanner input = new Scanner(System.in);
    System.out.print("digite um numero: ");
    int a = input.nextInt();
    
    System.out.print("digite um segundo numero: ");
    int b = input.nextInt();
    
    System.out.println("escolha uma opção:\n 1-somar\n 2-multiplicar\n 3-subtrair\n 4-dividir\n 5-parar");
    int op = input.nextInt();
    while (op!=5){
	    if(op==1){
	      int soma = a+b;
	      System.out.printf("esses numeros somados é equivalente a %d\n",soma);
	    }
	    else if(op==2){
	      int mult = a*b;
	      System.out.printf(" os numeros multiplicados são equivalentes a %d\n",mult);
	    }
	    else if (op==3){
	      int sub = a-b;
	      System.out.printf("a subtração desses numeros é equivalente a %d\n ", sub);
	    }
	    else if (op==4){
	      int div = a/b;
	      System.out.printf("a divisão desses numeros é igual a %d\n",div);
	    }
	    else if(op==5){
	      break;
      
	    }
	    else{
	      System.out.println("por favor digite um opção válida");
	    }
	    System.out.println("escolha uma opção:\n 1-somar\n 2-multiplicar\n 3-subtrair\n 4-dividir\n 5-parar");
	    op = input.nextInt();
	    }
	}
}

File: calculadora.py
print("questão da calculadora em python")
a=int(input("digite o primeiro numero: "))
b=int(input("digite o segundo numero: "))
op=input("digite uma opção\n 1- para fazer uma multiplicação\n 2- para soma\n 3-para subtrair\n 4-para dividir\n 5-para parar o programa")
while op!='5':
  if op=='1':
    mult=a*b
    print(mult)
  elif op=='2':
    soma=a+b
    print(soma)
  elif op=='3':
    sub= a-b
    print(sub)
  elif op=='4':
    div=a/b
    print(div)
  else:
    print('POR FAVOR DIGITE UMA OPÇÃO VALIDA.')
  op=input("digite uma opção\n 1- para fazer uma multiplicação\n 2- para soma\n 3-para subtrair\n 4-para dividir\n 5-para parar o programa")
